cloudbuild.yaml

```diff
@@ -12,7 +12,7 @@ steps:
       - importer-image-push
       - helm-chart-push
     env:
-      - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
+      - IMAGE_REGISTRY=us-central1-docker.pkg.dev/k8s-staging-images
```

Comment:
This needs to be `us-central1-docker.pkg.dev/k8s-staging-images/kueue`

Reply:
Not sure if we need to add `kueue` to the image registry, as the current image name is `$(IMAGE_REGISTRY)/kueue` which matches the registry name already.

Reply:
correct, this is the name of the registry, not the repo.

Reply:
Changed to `us-central1-docker.pkg.dev/k8s-staging-images/kueue`.